docs/pages/eas/hosting/reference/worker-runtime.mdx

```diff
@@ -16,54 +16,76 @@ For more information on how Workers work, see [Cloudflare Workers](https://devel
+| `node:querystring`         | <YesIcon />   |                                                                                    |
+| `node:readline`            | <NoIcon />    | Provided as non-functional JS stubs, since workers have no `stdin`                 |
+| `node:readline/promises`   | <NoIcon />    | Provided as non-functional JS stubs, since workers have no `stdin`                 |
+| `node:stream`              | <YesIcon />   |                                                                                    |
+| `node:stream/consumers`    | <YesIcon />   |                                                                                    |
+| `node:stream/promises`     | <YesIcon />   |                                                                                    |
+| `node:stream/web`          | <YesIcon />   |                                                                                    |
+| `node:string_decoder`      | <YesIcon />   |                                                                                    |
+| `node:timers`              | <YesIcon />   |                                                                                    |
+| `node:timers/promises`     | <YesIcon />   |                                                                                    |
+| `node:tls`                 | <NoIcon />    | Provides JS stubs but is unimplemented                                             |
+| `node:trace_events`        | <AlertIcon /> | Provided as non-functional JS stubs                                                |
+| `node:tty`                 | <YesIcon />   | Provided as JS shims redirecting output to the Console API                         |
+| `node:url`                 | <YesIcon />   |                                                                                    |
+| `node:util`                | <YesIcon />   |                                                                                    |
+| `node:util/types`          | <YesIcon />   |                                                                                    |
+| `node:worker_threads`      | <NoIcon />    | Provided as non-functional JS stubs, since workers don't support threading         |
+| `node:zlib`                | <YesIcon />   |                                                                                    |
+
+These modules generally provide a lower-accuracy polyfill or approximation of their Node.js counterparts.
+For example, the `http` and `https` modules only provide thin Node.js compatibility wrappers around the `fetch` API and cannot be used to make arbitrary HTTP requests.
+
+Any of the listed Node.js modules can be used in API routes or dependencies of your API routes as usual and will use appropriate compatibility modules. However, some of these modules may not provide any practical functionality and only exist to shim APIs to prevent runtime crashes.
```

Comment:
```suggestion
Any of the above listed Node.js modules can be used in API routes or dependencies of your API routes as usual and will use appropriate compatibility modules. However, some of these modules may not provide any practical functionality and only exist to shim APIs to prevent runtime crashes.
```

Let's make it clearer by indicating where the Node.js modules are listed.